packages/daemon/test/test-endo.js

```diff
@@ -979,8 +1034,8 @@ test('name and reuse inspector', async t => {
   await stop(locator);
 });
 
-// Regression test for: https://github.com/endojs/endo/issues/2021
-test('eval-mediated worker name', async t => {
+// Regression test for https://github.com/endojs/endo/issues/2021
+test.failing('eval-mediated worker name', async t => {
```

Comment:
So, we have whacked a mole?

Reply:
I am consulting my exterminator as to our next steps.